docs/node-template-release.md

```diff
@@ -0,0 +1,76 @@
+# Node Template Release Process
+
+1. It has to be in a github checkout directory with your current work committed into
+`https://github.com/paritytech/substrate/`, because this is where the build script will check out
+from. Assume you are in root directory of Substrate. Run:
```

Comment:
this is confusing to me, can you rephrase it?

Reply:
@apopiak this paragraph is updated.